packages/client/src/NetworkNodeFacade.ts

```diff
@@ -68,21 +70,25 @@ export class NetworkNodeFactory {
 export class NetworkNodeFacade {
 
     private cachedNode?: NetworkNodeStub
+    private cachedEntryPoints?: NetworkPeerDescriptor[]
```

Comment:
Caching is maybe not needed. The data is only queried once to initialize the node, and we never need to re-initialize the node again.